pkg/apis/autoscaling/annotation_validation_test.go

```diff
@@ -17,95 +17,118 @@ limitations under the License.
-			Paths:   []string{MaxScaleAnnotationKey},
-		},
+		expectErr:   "expected 1 <= -1 <= 2147483647: autoscaling.knative.dev/maxScale",
 	}, {
 		name:        "minScale is foo",
 		annotations: map[string]string{MinScaleAnnotationKey: "foo"},
-		expectErr: &apis.FieldError{
-			Message: fmt.Sprintf("Invalid %s annotation value: must be an integer equal or greater than 0", MinScaleAnnotationKey),
-			Paths:   []string{MinScaleAnnotationKey},
-		},
+		expectErr:   "expected 1 <= foo <= 2147483647: autoscaling.knative.dev/minScale",
 	}, {
 		name:        "maxScale is bar",
 		annotations: map[string]string{MaxScaleAnnotationKey: "bar"},
-		expectErr: &apis.FieldError{
-			Message: fmt.Sprintf("Invalid %s annotation value: must be an integer equal or greater than 0", MaxScaleAnnotationKey),
-			Paths:   []string{MaxScaleAnnotationKey},
-		},
+		expectErr:   "expected 1 <= bar <= 2147483647: autoscaling.knative.dev/maxScale",
+	}, {
+		name:        "max/minScale is bar",
+		annotations: map[string]string{MaxScaleAnnotationKey: "bar", MinScaleAnnotationKey: "bar"},
+		expectErr:   "expected 1 <= bar <= 2147483647: autoscaling.knative.dev/maxScale, autoscaling.knative.dev/minScale",
 	}, {
 		name:        "minScale is 5",
 		annotations: map[string]string{MinScaleAnnotationKey: "5"},
-		expectErr:   nil,
 	}, {
 		name:        "maxScale is 2",
 		annotations: map[string]string{MaxScaleAnnotationKey: "2"},
-		expectErr:   nil,
 	}, {
 		name:        "minScale is 2, maxScale is 5",
 		annotations: map[string]string{MinScaleAnnotationKey: "2", MaxScaleAnnotationKey: "5"},
-		expectErr:   nil,
 	}, {
 		name:        "minScale is 5, maxScale is 2",
 		annotations: map[string]string{MinScaleAnnotationKey: "5", MaxScaleAnnotationKey: "2"},
-		expectErr: &apis.FieldError{
-			Message: fmt.Sprintf("%s=%v is less than %s=%v", MaxScaleAnnotationKey, 2, MinScaleAnnotationKey, 5),
-			Paths:   []string{MaxScaleAnnotationKey, MinScaleAnnotationKey},
-		},
+		expectErr:   "autoscaling.knative.dev/maxScale=2 is less than autoscaling.knative.dev/minScale=5: autoscaling.knative.dev/maxScale, autoscaling.knative.dev/minScale",
 	}, {
 		name: "minScale is 0, maxScale is 0",
 		annotations: map[string]string{
 			MinScaleAnnotationKey: "0",
 			MaxScaleAnnotationKey: "0",
 		},
-		expectErr: nil,
+	}, {
+		name:        "panic window percentange bad",
+		annotations: map[string]string{PanicWindowPercentageAnnotationKey: "-1"},
+		expectErr:   "expected 1 <= -1 <= 100: autoscaling.knative.dev/panicWindowPercentage",
+	}, {
+		name:        "panic window percentange bad2",
+		annotations: map[string]string{PanicWindowPercentageAnnotationKey: "202"},
+		expectErr:   "expected 1 <= 202 <= 100: autoscaling.knative.dev/panicWindowPercentage",
+	}, {
+		name:        "panic window percentange bad3",
+		annotations: map[string]string{PanicWindowPercentageAnnotationKey: "fifty"},
+		expectErr:   "invalid value: fifty: autoscaling.knative.dev/panicWindowPercentage",
+	}, {
```

Comment:
Do you want a "panic window percentage good" to go with "panic threshold percentage good"?

Reply:
Corrected. Thanks.